packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -1171,27 +1170,7 @@ export class WorkflowExecute {
 			// For webhook nodes always simply pass the data through
 			return { data: inputData.main as INodeExecutionData[][] };
 		} else {
-			// For nodes which have routing information on properties
-
-			const routingNode = new RoutingNode(
-				workflow,
-				node,
-				connectionInputData,
-				runExecutionData ?? null,
-				additionalData,
-				mode,
-			);
-
-			return {
-				data: await routingNode.runNode(
-					inputData,
-					runIndex,
-					nodeType,
-					executionData,
-					undefined,
-					abortSignal,
-				),
-			};
+			throw new ApplicationError('Declarative nodes should been handled as regular nodes');
```

Comment:
Will this break usage of declarative nodes as regular nodes? As they'll still do not have the execute method.

Reply:
they should all have an `execute` method at this point. sounds like we might have a bug.